blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,9 @@ Unreleased Changes
 Changes
 =======
 
+- Increased the version of admin-ui to 1.15.1 which has the queries adapted to
```

Comment:
This should go under fixes. Also, I'd omit the `Increased the version..` detail, how the problem was fixed is not relevant to the user, nor is it relevant for users that the admin-ui is separately versioned. 

From a user perspective what was broken is that partitions didn't show up in the table view of the admin-ui